package.json

```diff
@@ -2,113 +2,132 @@
   "name": "@readme/markdown",
   "description": "ReadMe's React-based Markdown parser",
   "author": "Rafe Goldberg <rafe@readme.io>",
-  "version": "6.87.1",
+  "version": "6.75.0-beta.77",
```

Comment:
not sure how the package publishing stuff works here but should all of this qualify for a major v7 release since i assume it's all a pretty big breaking change?

Reply:
Yes definitely, forgot to do that.

Reply:
Ok, updated the commit body with `BREAKING CHANGE:`, hopefully that changes the major number 🤞🏼